function testcase ( ) { try { } catch ( e ) { return e instanceof SyntaxError ; } } runTestCase ( testcase ) ;